dev-packages/node-integration-tests/suites/tracing/meta-tags/test.ts

```diff
@@ -16,8 +16,7 @@ describe('getTraceMetaTags', () => {
       baggage: 'sentry-environment=production',
     });
 
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
+    // @ts-expect-error - response is defined, types just don't reflect it
```

Comment:
wait but is this not introducing a ts-expect-error?

Reply:
lol thx good catch! I guess I changed it back to expect-error to test the rule and the rule was incorrect so it actually still passed 🤦‍♂️

Reply:
Anyway, I adjusted the rule, now it actually throws when you use `@ts-expect-error`